p2p/src/authenticated/discovery/actors/tracker/actor.rs

```diff
@@ -94,64 +87,30 @@ impl<E: Spawner + Rng + Clock + GClock + RuntimeMetrics, C: Signer> Actor<E, C>
             releaser,
         );
 
+        // Create peer validator
```

Comment:
We return this from `tracker` to avoid leaking configuration knowledge to all other actors.